crates/sui-framework/packages/deepbook/sources/clob_v2.move

```diff
@@ -248,6 +248,25 @@ module deepbook::clob_v2 {
         quote_asset_trading_fees: Balance<QuoteAsset>,
     }
 
+    /// Capability granting permission to access an entry in `Pool.quote_asset_trading_fees`.
+    struct PoolOwnerCap has key, store {
+        id: UID,
+        /// The owner of this AccountCap. Note: this is
+        /// derived from an object ID, not a user address
+        owner: address
+    }
+
+    /// Function to withdraw fees created from a pool
+    public fun withdraw_fees<BaseAsset, QuoteAsset>(
+        pool: &mut Pool<BaseAsset, QuoteAsset>,
+        _pool_owner_cap: &PoolOwnerCap,
+        ctx: &mut TxContext,
+    ): Coin<QuoteAsset> {
+        let quanity = quote_asset_trading_fees_value(pool);
+        let to_withdraw = balance::split(&mut pool.quote_asset_trading_fees, quanity);
+        coin::from_balance(to_withdraw, ctx)
```

Comment:
I'm curious what @damirka thinks re best practices for returning a `Coin` vs a `Balance` here. My instinct would be to return the more general `Balance` type and let the caller wrap it into a `Coin` if needed, but maybe this is better for some common PTB patterns (e.g., merging back into the gas coin)?